sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosItemTest.java

```diff
@@ -198,6 +199,31 @@ public void queryItems() throws Exception{
         assertThat(feedResponseIterator3.iterator().hasNext()).isTrue();
     }
 
+    @Test(groups = { "simple" }, timeOut = TIMEOUT)
+    public void queryItemsWithCustomCorrelationActivityId() throws Exception{
```

Comment:
Was this validated e2e where the correlated activity id was in the kusto logs?

Reply:
The funny thing is that 2 bytes are changed in order betweenSystem.Net Guid and UUID - but that is the case for some other properties as well - seems to be something worth discussing (looks like a difference in Java vs. .Net)

"b9cd8180-4f89-4e4d-3a8a-9c1264a269d4" in .Net/Kusto is "b9cd8180-4f89-4e4d-**_8a3a_**-9c1264a269d4" in java logs

Reply:
Fixed teh incorrect UUID encoding (was only used for ActivityId and CorrelatedActivityId - so safe change